nestkernel/conn_builder.h

```diff
@@ -198,17 +209,10 @@ class ConnBuilder
   std::vector< std::shared_ptr< WrappedThreadException > > exceptions_raised_;
 
   // Name of the pre synaptic and postsynaptic elements for this connection builder
-  Name pre_synaptic_element_name_;
-  Name post_synaptic_element_name_;
+  std::string pre_synaptic_element_name_;
+  std::string post_synaptic_element_name_;
```

Comment:
These seem SP-specific. Do we need them in the base class?

Reply:
I initially also thought so and even tried moving them. However, they are used in `ConnBuilder::change_connected_synaptic_elements()`, which gets called by `sp_disconnect()` in each and every `ConnBuilder` child class.

Maybe this can be further disentangled in a future PR? I would regard this as outside the scope of this on. See also #2311 for a similar effort to split SP related code from general NEST base functionality.